### PR TITLE
#167190406 View notifications

### PR DESCRIPTION
**Pull request by cvjude**
#### What does this PR do?
- Allow's Users to view Notifications

#### How can this be manually tested?
- Clone the repo
- Run the command below in your terminal to install all dependencies
```
npm install
``` 
- Run the command below to test
```
npm test
```

#### Any background context you want to provide?
none

#### What are the relevant Pivotal Tracker stories? 
- [#167190406](https://www.pivotaltracker.com/story/show/167190406)

### CODE REVIEW

File: src/actions/notifications.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+export const markAllRead = () => async dispatch => {
+  try {
+    const response = await axiosInstance.patch('notifications/read');
+    dispatch({
+      type: types.MARK_READ,
+      payload: response.data,
+    });
+  } catch (err) {
+    // eslint-disable-next-line no-console
+    // console.log(err);
```
Comment (Lundii) [inline src/actions/notifications.js:41]: Nice work Jude. I like the UI design. I am just thinking, if you are not expecting an error in this API call, is there a need for using the try and catch block?

File: src/actions/notifications.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  try {
+    const response = (type) ? await axiosInstance.patch('notifications/email/subscribe')
+      : await axiosInstance.delete('notifications/email/subscribe');
+    dispatch({
+      type: types.UPDATE_SUB,
+      payload: response.data,
+    });
+  } catch (err) {
+    // eslint-disable-next-line no-console
+    // console.log(err);
```
Comment (Lundii) [inline src/actions/notifications.js:55]: Same here. Thank you.

**Review by obayomi96 — APPROVED** [review]
Awesome work @cvjude , you have done a magnificent job on all aspects of this feature. Well done

File: src/actions/notifications.js (hunk 1 above)
Comment (cvjude) [reply src/actions/notifications.js:41]: Yes, I wrapped in a try and catch block to avoid the App from crashing in the advent of an error.

Comment (Lundii) [reply src/actions/notifications.js:41]: Alright. Nice

**Review by Lundii — APPROVED** [review]

File: src/reducers/__test__/reducers.spec.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
+// import { combineReducers } from 'redux';
```
Comment (chuxmykel) [inline src/reducers/__test__/reducers.spec.js]: Hey @cvjude. Is there a reason why these lines were commented out? Also if they are not useful, I'd suggest we take them out entirely.

**Review by chuxmykel — APPROVED** [review]
Nice work @cvjude. Looks good to me.

**Review by Hadeneekeh — APPROVED** [review]
Great job with the notification dropdown styling!
LGTM!!!

**Review by encodedBicoding — APPROVED** [review]
Great job using pusher. LGTM

**Pull request merged**